.github/workflows/ci.yaml

```diff
@@ -229,7 +229,7 @@ jobs:
             SANDBOX_ID="STAGING_SANDBOX_ID"
             CHART_REF="STAGING_CHART_REF"
             KUBERNETES_ENV="staging"
-            REPLICA_COUNT="2"
+            REPLICA_COUNT="1"
```

Comment:
we can change this to 0 (it's the staging envs and they will be scaled back anyway)